travis/build.sh

```diff
@@ -11,11 +11,11 @@ if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then
 fi
 
 case "${TRAVIS_BRANCH}" in
-    "next")
+    "specify-credstash-version")
```

Comment:
Do we need this? I thought the above line was for only for testing.

Reply:
Yes, it is for testing and will be removed later. I am trying to deploy it to TEST site without merging.